style.css

```diff
@@ -278,3 +282,205 @@ footer {
   gap: 20px;
   font-size: 22px;
 }
+
+.hide-mobile {
+  display: none;
+}
+
+.hide-desktop {
+  display: block;
+}
+
+.fa-envelope {
+  display: none !important;
+}
+
+@media screen and (min-width: 768px) {
+  .fa-bars {
+    display: none !important;
+  }
+
+  .fa-envelope {
+    display: block !important;
+  }
+
+  .hide-mobile {
+    display: block;
+  }
+
+  .hide-desktop {
+    display: none;
+  }
+
+  nav {
+    display: flex;
+    gap: 20px;
+  }
+
+  header {
+    padding: 20px 70px;
+  }
+
+  .landing {
+    height: 90vh;
+    display: flex;
+    flex-direction: row;
+
+    /* justify-content: space-around; */
+    padding: 200px 50px;
+    background-color: #1c1a19;
+    color: white;
+    background-image: url("resources/Illustration-header.png");
+    background-repeat: no-repeat;
+    background-position: right;
+  }
+
+  .icons {
+    display: flex;
+    flex-direction: column;
+    margin-top: 80px;
+    margin-right: 50px;
+    gap: 10px;
+    font-size: 25px;
+  }
+
+  .hey {
+    width: 500px;
+  }
+
+  h1 {
+    color: #ff6b00;
+    font-size: 72px;
+    font-family: "Crete Round", serif;
+  }
+
+  .roboto {
+    font-family: "Roboto", sans-serif;
+    font-size: 32px;
+  }
+
+  .description {
+    font-family: "Poppins", sans-serif;
+    font-size: 20px;
+  }
+
+  .sec2 {
+    position: relative;
+  }
+
+  .underline2 {
+    position: absolute;
+    left: 100%;
+    bottom: 50%;
+    width: 750px;
+  }
+
+  .card1 {
+    display: flex;
+    gap: 10;
+  }
+
+  .sec3 {
+    padding: 0 200px;
+  }
+
+  .card1 > img {
+    width: 500px;
+  }
+
```

Comment:
- [ ] In your design, kindly use the right spacing and sizing of all the elements so that it is the same as the Figma design. Ensure your text wraps like that of the design and please use the given font family.

### Given design
![Screenshot 2023-08-10 at 20 14 55 (2)](https://github.com/JimOluwaseyi/myPortfolio/assets/64774936/f1e62911-afd9-4d8e-b9f2-91e127c0b8ab)

### Your design
<img width="1116" alt="Screenshot 2023-08-10 at 20 17 35" src="https://github.com/Wisdomibu/Portfolio/assets/64774936/c28f5769-4c85-4f7b-82e0-ee49c07a8cd2">

- [ ] In this section, the first image in the given design is how your boxes should look when the cursor hovers on it and it should look like the others when the cursor is removed.

### Given design
![Screenshot 2023-08-10 at 20 15 08 (2)](https://github.com/JimOluwaseyi/myPortfolio/assets/64774936/54479951-9896-4e46-89ce-ded4fc030514)


### Your design
<img width="1068" alt="Screenshot 2023-08-10 at 20 17 43" src="https://github.com/Wisdomibu/Portfolio/assets/64774936/d6cdbdc4-1154-4705-b0ae-cf5f10643912">